app/views/ops/_analytics_details_tab.html.haml

```diff
@@ -1,8 +1,9 @@
 - if @sb[:active_tab] == "analytics_details"
   - if @sb[:analytics_rpt]
     %fieldset
-      %h3= @sb[:analytics_rpt].title
-      - if !@sb[:analytics_rpt].table.nil? || @sb[:analytics_rpt].table.data.empty?
+      %h3
+        = @sb[:analytics_rpt].title
+      - if !@sb[:analytics_rpt].table.present? || !@sb[:analytics_rpt].table.data.empty?
```

Comment:
Seems like you changed the first half of the expression from `!@sb[:analytics_rpt].table.nil?` to `!@sb[:analytics_rpt].table.present?`.  It seems that `not nil?` should be changed to `present?` or `@sb[:analytics_rpt].table.present?`